src/main/java/com/google/gerrit/extensions/common/IncludedInInfo.java

```diff
@@ -0,0 +1,11 @@
+package com.google.gerrit.extensions.common;
```

Comment:
Please move this to `com.google.gerrit.extensions.api.changes` and copy the official file 1:1:
https://github.com/gerrit-review/gerrit/blob/master/gerrit-extension-api/src/main/java/com/google/gerrit/extensions/api/changes/IncludedInInfo.java